### PR TITLE
doc: nRF Desktop final style review

### PR DESCRIPTION
**Pull request by greg-fer**
Apply edits listed in DESK-685 and last fixes.

Signed-off-by: Grzegorz Ferenc <Grzegorz.Ferenc@nordicsemi.no>

### CODE REVIEW

**Comment by greg-fer** [comment]
@ru-fu , despite its name, this review is not yet final. There will be at least one more PR from Pawel.

File: applications/nrf_desktop/doc/ble_latency.rst [hunk 1]
```diff
@@ -1,42 +1,61 @@
...
+
 The ``ble_latency`` module receives :ref:`nrf_desktop_config_channel` events, but it is not configurable with the :ref:`nrf_desktop_config_channel`.
 The module does not register itself using the ``GEN_CONFIG_EVENT_HANDLERS`` macro.
 
-If the nrfxlib Link Layer (:option:`CONFIG_BT_LL_NRFXLIB`) is selected on the connected Bluetooth central, the connection latency can be controlled only on the central side. (The peripheral's requests to modify the connection latency are ignored.)
+.. note::
+   Zephyr's :ref:`zephyr:bluetooth` API does not allow to use the LLPM connection intervals in the connection parameter update request.
+   If the LLPM connection interval is in use:
+
+   * The nRF Desktop peripheral uses a 7.5-ms interval in the request.
```
Comment (pdunaj) [inline applications/nrf_desktop/doc/ble_latency.rst:56]: why `-` in `7.5-ms`?

**Comment by pdunaj** [comment]
@greg-fer there are some issues in CI
```
nrf/rst/doc/nrf/applications/nrf_desktop/doc/dev_descr.rst:20: WARNING: Inline literal start-string without end-string.
```

File: applications/nrf_desktop/README.rst [hunk 2]
```diff
@@ -43,10 +61,12 @@ The event-based communication for each application module is presented as a tabl
 |                                       |             |                                    | ``eventC``   | The module that reacts on ``eventC``.   |
 +---------------------------------------+-------------+------------------------------------+--------------+-----------------------------------------+
 
-The table contains the following columns:
+Each module event table contains the following columns:
 
 Source Module
    The module that submits a given application event.
+   Some of these modules can have many listeners or sources.
```
Comment (MarekPieta) [inline applications/nrf_desktop/README.rst:68]: `Some of these events`

File: applications/nrf_desktop/README.rst [hunk 3]
```diff
@@ -60,40 +80,29 @@ Output Event
 
 Sink Module
    The module that reacts on the application event.
+   Some of these modules can have many listeners or sources.
```
Comment (MarekPieta) [inline applications/nrf_desktop/README.rst:83]: `Some of these events`

File: applications/nrf_desktop/README.rst [hunk 4]
```diff
@@ -148,77 +157,81 @@ The following threads are kept running in the application:
...
    nRF Desktop mouse HID data sensing and transmission
 
+----
+
 Requirements
 ************
 
-The project supports several boards related to the following hardware reference designs.
-Depending on what board you use, you need to select a respective configuration file and a :ref:`build type <nrf_desktop_requirements_build_types>`.
+The nRF Desktop application supports several development kits related to the following hardware reference designs.
```
Comment (MarekPieta) [inline applications/nrf_desktop/README.rst:233]: Development kit or board ?

File: applications/nrf_desktop/README.rst [hunk 5]
```diff
@@ -282,35 +296,40 @@ Configuration files are provided for different build types for each supported bo
...
-The following build types are available in nRF Desktop:
+The following build types are available for various boards in nRF Desktop:
 
 * ``ZRelease`` -- Release version of the application with no debugging features.
+* ``ZReleaseB0`` -- ``ZRelease`` build type with the support for the B0 bootloader enabled (for :ref:`background DFU <nrf_desktop_bootloader_background_dfu>`).
+* ``ZReleaseMCUBoot`` -- ``ZRelease`` build type with the support for the MCUBoot bootloader enabled (for :ref:`serial recovery DFU <nrf_desktop_bootloader_serial_dfu>`).
 * ``ZDebug`` -- Debug version of the application; the same as the ``ZRelease`` build type, but with debug options enabled.
-* ``ZDebugWithShell`` -- ``ZDebug`` build type with the shell enabled.
-* ``ZReleaseB0`` -- ``ZRelease`` build type with the support for the bootloader enabled.
 * ``ZDebugB0`` -- ``ZDebug`` build type with the support for the bootloader enabled.
```
Comment (MarekPieta) [inline applications/nrf_desktop/README.rst:305]: I would mention that this is B0 bootloader (to be consistent with realease configurations description)

File: applications/nrf_desktop/README.rst [hunk 6]
```diff
@@ -282,35 +296,40 @@ Configuration files are provided for different build types for each supported bo
...
+* ``ZDebugMCUBoot`` -- ``ZDebug`` build type with the support for the MCUBoot bootloader enabled (for :ref:`serial recovery DFU <nrf_desktop_bootloader_serial_dfu>`).
+* ``ZDebugWithShell`` -- ``ZDebug`` build type with the shell enabled.
 
-In nRF Desktop, not every board can support every build type.
-If the given build type is not supported on the selected board, an error message will appear when `Building and running`_.
-For example, if the ``ZDebugWithShell`` build type is not supported on the selected board, the following notification appears:
+In nRF Desktop, not every development kit can support every build type mentioned above.
```
Comment (MarekPieta) [inline applications/nrf_desktop/README.rst:309]: board ?

File: applications/nrf_desktop/README.rst [hunk 7]
```diff
@@ -703,7 +732,7 @@ The nRF Desktop application is built in the same way to any other NCS applicatio
 Selecting a build type
 ======================
 
-Before you start testing the application, you can select one of the :ref:`nrf_desktop_requirements_build_types`, depending on your board and building method.
+Before you start testing the application, you can select one of the :ref:`nrf_desktop_requirements_build_types`, depending on your development kit and building method.
```
Comment (MarekPieta) [inline applications/nrf_desktop/README.rst:735]: board ?

File: applications/nrf_desktop/README.rst [hunk 8]
```diff
@@ -860,28 +895,33 @@ Moreover, keep the default ``ZDebug`` build type that the application is built w
 .. note::
     The following procedure uses the gaming mouse configuration as an example.
 
-To use the nRF Desktop project with your custom board:
+To use the nRF Desktop application with your custom board:
 
-1. Define the board by copying the nRF Desktop board files that are the closest match for your hardware.
-   For example, for gaming mouse use ``nrf/boards/arm/nrf52840gmouse_nrf52840``.
+1. Define the reference design by copying the nRF Desktop reference design files that are the closest match for your hardware.
```
Comment (MarekPieta) [inline applications/nrf_desktop/README.rst:900]: board or reference design in this paragraph?
Zephyr uses `board` for that case

File: applications/nrf_desktop/README.rst [hunk 9]
```diff
@@ -1085,46 +1127,58 @@ Optionally, you can also enable the following module:
...
-The B0 is a small, simple and secure bootloader that allows the application to boot directly from one of the application slots, thus increasing the speed of the direct firmware upgrade (DFU) process.
-More information about the B0 can be found at :ref:`bootloader`.
 
 Enabling the bootloader
-    To enable the B0 bootloader, select the :option:`CONFIG_SECURE_BOOT` Kconfig option.
-    If this option is not selected, the application will be built without the bootloader, and the DFU will not be available.
+-----------------------
+
+To enable the B0 bootloader, select the :option:`CONFIG_SECURE_BOOT` Kconfig option.
+If this option is not selected, the application will be built without the bootloader, and the background DFU will not be available.
```
Comment (MarekPieta) [inline applications/nrf_desktop/README.rst:1149]: `without the B0 bootloader` (there is also MCUBoot)

File: applications/nrf_desktop/doc/ble_latency.rst (hunk 1 above)
Comment (greg-fer) [reply applications/nrf_desktop/doc/ble_latency.rst:56]: Discussed on Teams.

File: applications/nrf_desktop/doc/ble_scan.rst [hunk 10]
```diff
@@ -96,28 +101,29 @@ The |ble_scan| module stores the following information for every bonded peer:
...
+   The initial Bluetooth connection interval is set by default to 7.5 ms, that is to the shortest connection interval allowed by the Bluetooth specification.
+#. The connection is secured.
+#. The peer discovery is started.
+#. After the :ref:`nrf_desktop_ble_discovery` completes the peer discovery, the :ref:`nrf_desktop_ble_conn_params` receives the ``ble_discovery_complete_event`` and updates the Bluetooth connection interval.
 
-   .. note::
-      If a Bluetooth peer is aready connected with a 1-ms connection interval, the next peer is connected with a 10-ms connection interval instead of 7.5 ms.
-      This is required to avoid Bluetooth scheduling issues.
+.. note::
+   If a Bluetooth peer is aready connected with a 1-ms connection interval, the next peer is connected with a 10-ms connection interval instead of 7.5 ms.
```
[the remark this answers is not in the document]
Comment (greg-fer) [reply applications/nrf_desktop/doc/ble_scan.rst:116]: Discussed on Teams.

File: applications/nrf_desktop/doc/ble_scan.rst [hunk 11]
```diff
@@ -96,28 +101,29 @@ The |ble_scan| module stores the following information for every bonded peer:
...
+   This is required to avoid Bluetooth scheduling issues.
 
-   At this point, the scanning can be restarted.
+At this point, the scanning can be restarted.
 
 Connection parameter update request
-   The module handles Zephyr's Bluetooth LE connection parameter update request:
```
[the remark this answers is not in the document]
Comment (greg-fer) [reply applications/nrf_desktop/doc/ble_scan.rst:118]: Section removed.

File: applications/nrf_desktop/README.rst (hunk 2 above)
Comment (greg-fer) [reply applications/nrf_desktop/README.rst:68]: Fixed.

File: applications/nrf_desktop/README.rst (hunk 3 above)
Comment (greg-fer) [reply applications/nrf_desktop/README.rst:83]: Fixed.

File: applications/nrf_desktop/doc/battery_charger.rst [hunk 12]
```diff
@@ -3,53 +3,56 @@
...
   * ``CONFIG_DESKTOP_BATTERY_CHARGER_CSO_PULL_NONE`` - CSO pin pull disabled (default setting).
   * ``CONFIG_DESKTOP_BATTERY_CHARGER_CSO_PULL_UP`` - CSO pin pull up.
   * ``CONFIG_DESKTOP_BATTERY_CHARGER_CSO_PULL_DOWN`` - CSO pin pull down.
 
-* ``CONFIG_DESKTOP_BATTERY_CHARGER_ENABLE_PIN`` - pin that enables the battery charging.
-* ``CONFIG_DESKTOP_BATTERY_CHARGER_ENABLE_INVERSED`` - option inversing charging enable signal.
-* ``CONFIG_DESKTOP_BATTERY_CHARGER_CSO_FREQ`` - CSO pin state switching frequency for when a charging error occurrs (in Hz).
+* ``CONFIG_DESKTOP_BATTERY_CHARGER_ENABLE_PIN`` - Configures the pin that enables the battery charging.
+* ``CONFIG_DESKTOP_BATTERY_CHARGER_ENABLE_INVERSED`` - Option for inversing the charging enable signal.
+* ``CONFIG_DESKTOP_BATTERY_CHARGER_CSO_FREQ`` - Sets the CSO pin state switching frequency for when a charging error occurs (in Hz).
```
[the remark this answers is not in the document]
Comment (greg-fer) [reply applications/nrf_desktop/doc/battery_charger.rst:37]: Fixed.

File: applications/nrf_desktop/README.rst (hunk 4 above)
Comment (greg-fer) [reply applications/nrf_desktop/README.rst:233]: Fixed.

File: applications/nrf_desktop/README.rst (hunk 5 above)
Comment (greg-fer) [reply applications/nrf_desktop/README.rst:305]: Fixed.

File: applications/nrf_desktop/README.rst (hunk 8 above)
Comment (greg-fer) [reply applications/nrf_desktop/README.rst:900]: We will solve this in Pawel's PR. Need to double check.

File: applications/nrf_desktop/README.rst (hunk 9 above)
Comment (greg-fer) [reply applications/nrf_desktop/README.rst:1149]: Fixed.

**Comment by thst-nordic** [comment]
please ignore the 'CI/Jenkins/NRF/ZEPHYR' status.   The CI posted it by mistake yesterday.

**Pull request merged**